### PR TITLE
Replace theme conditionals in templates with new skyIfTheme directive

### PR DESCRIPTION
**Pull request by johnhwhite**
- Replace theme conditionals in templates with new skyIfTheme directive

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-replace-conditionals-with-skyiftheme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-replace-conditionals-with-skyiftheme)

File: src/app/visual/inline-form/inline-form-demo-tile.component.html [hunk 1]
```diff
@@ -31,9 +31,8 @@
 
       <ng-template #tileForm>
         <sky-input-box
-          [ngClass]="{
-            'sky-margin-stacked-lg':
-              (themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'modern'
+          [skyThemeClass]="{
```
Comment (Blackbaud-AlexKingman) [inline src/app/visual/inline-form/inline-form-demo-tile.component.html:34]: Can we remove `SkyThemeService` from the constructor in the `.ts` file?

File: src/app/visual/inline-form/inline-form-demo.component.html [hunk 2]
```diff
@@ -22,9 +22,8 @@ <h2>"Add" mode (default)</h2>
     #addForm
   >
     <sky-input-box
-      [ngClass]="{
-        'sky-margin-stacked-lg':
-          (themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'modern'
+      [skyThemeClass]="{
```
Comment (Blackbaud-AlexKingman) [inline src/app/visual/inline-form/inline-form-demo.component.html:25]: Can we remove `SkyThemeService` from the constructor in the `.ts` file?

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
Two things...

File: src/app/visual/inline-form/inline-form-demo.component.html (hunk 2 above)
Comment (johnhwhite) [reply src/app/visual/inline-form/inline-form-demo.component.html:25]: In the visual demo, `SkyThemeService` is used for the `<sky-docs-demo-control-panel-theme>` theme switcher.

Comment (Blackbaud-AlexKingman) [reply src/app/visual/inline-form/inline-form-demo.component.html:25]: Good call. But can we still remove the service from the `inline-form-demo-tile.component.ts` file?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-replace-conditionals-with-skyiftheme)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**